src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTokenExtensions.vb

```diff
@@ -57,6 +57,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsGlobalStatementContext(token As SyntaxToken, position As Integer) As Boolean
```

Comment:
This seems to be a fairly specialized (vs general purpose) method and I can only find one caller.  Perhaps it should be inlined or made a private method of the caller's type?